x-pack/plugin/identity-provider/src/test/resources/rest-api-spec/api/idp.saml_get_metadata.json

```diff
@@ -19,6 +19,9 @@
           }
         }
       ]
+    },
+    "params": {
+      "acs": { ... }
```

Comment:
My use of `...` was just a placeholder - I think this should be populated with something meaningful.

Reply:
Meh , I did a two step process of 1) all the suggestions make sense and later 2) lets merge these and forgot to take care of this . Will adjust